lib/lib-doi/src/main/java/org/sagebionetworks/doi/datacite/DataciteTranslator.java

```diff
@@ -0,0 +1,78 @@
+package org.sagebionetworks.doi.datacite;
+
+import org.apache.xerces.jaxp.DocumentBuilderFactoryImpl;
+import org.sagebionetworks.repo.model.doi.*;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataciteTranslator {
+
+	/*
+	 * Translates a Datacite XML String into a DoiMetadata object
+	 * Works for XML adherent to Datacite Schemas 2.2 and 4.1
+	 */
+	public DoiMetadata translate(String xml) {
+		xml.replaceAll("\n","");
+		Document dom = null;
+		try {
+			DocumentBuilder documentBuilder = DocumentBuilderFactoryImpl.newInstance().newDocumentBuilder();
+			dom = documentBuilder.parse(new InputSource(new StringReader(xml)));
+		} catch (ParserConfigurationException e) {
+			throw new RuntimeException("Error occurred while configuring XML parser", e);
+		} catch (SAXException | IOException e) {
+			throw new RuntimeException("Error occurred while parsing metadata", e);
+		}
+
+		// There may be multiple creators or titles
+		DoiMetadata doiMetadata = new DoiMetadata();
```

Comment:
List<DoiCreator> readCreators(Document dom) all creator details should be tested through this method.